installer/Gopkg.toml

```diff
@@ -0,0 +1,36 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+required = ["github.com/golang/dep", "github.com/GoASTScanner/gas", "github.com/go-yaml/yaml", "github.com/golang/lint/golint"]
```

Comment:
We're pulling in kubernetes-sigs/yaml in the provider project.  It's based on go-yaml.  Kubernetes projects seem to like using forked versions (into the k8s.io or sigs.k8s.io repos) of outside projects.

Reply:
I'm fine with pulling the same as long as they're compatible